src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -1862,7 +1862,7 @@
                       Grid.Row="2"
                       Grid.RowSpan="2"
                       Height="Auto"
-                      Width="3"
+                      Width="0"
```

Comment:
@QilongTang After removing the border, there was margin on the left due to this splitter element. I removed this for now, but let me know what you think with that. It was looking little odd due to the space on the left.

Reply:
Sounds good. Can you add a picture of the latest look?

Reply:
![new](https://user-images.githubusercontent.com/43763136/60458049-916e2980-9c0b-11e9-9e19-4f353c850983.PNG)

Reply:
Perfect!